### DISCUSSION TITLE
8.9.0-pre.1

### DISCUSSION THREAD

**Discussion by wasabii** [ikvmnet/ikvm#548] [Announcements]
## What's Changed
* Platform Specific IO, Security, Base Libs by @wasabii in https://github.com/ikvmnet/ikvm/pull/521
* Native AWT by @wasabii in https://github.com/ikvmnet/ikvm/pull/528
* README typo by @Sinan-Karakaya in https://github.com/ikvmnet/ikvm/pull/533
* Linux CI Build by @wasabii in https://github.com/ikvmnet/ikvm/pull/532
* Update to jdk8u192-b26 by @wasabii in https://github.com/ikvmnet/ikvm/pull/500
* win-x86 test suites by @wasabii in https://github.com/ikvmnet/ikvm/pull/456
* Upgrade to jdk8u275-b01 by @wasabii in https://github.com/ikvmnet/ikvm/pull/534
* Add osx-arm64 tests by @wasabii in https://github.com/ikvmnet/ikvm/pull/537

## New Contributors
* @Sinan-Karakaya made their first contribution in https://github.com/ikvmnet/ikvm/pull/533

**Full Changelog**: https://github.com/ikvmnet/ikvm/compare/8.8.1...8.9.0-pre.1

<hr /><em>This discussion was created from the release <a href='https://github.com/ikvmnet/ikvm/releases/tag/8.9.0-pre.1'>8.9.0-pre.1</a>.</em>

**Comment by LEI-Hongfaan** [comment]
NU1102: Unable to find package IKVM.MSBuild with version (>= 8.9.0-pre.1)
